pathd/path_pcep_controller.c

```diff
@@ -194,6 +201,7 @@ int pcep_ctrl_initialize(struct thread_master *main_thread,
 	ctrl_state->pcc_count = 0;
 	ctrl_state->pcc_opts =
 		XCALLOC(MTYPE_PCEP, sizeof(*ctrl_state->pcc_opts));
+	memset(ctrl_state->pcc, 0, sizeof(ctrl_state->pcc[0]) * MAX_PCC);
```

Comment:
The memory allocated with XCALLOC is already cleared so there is no need to do any memeset.

Reply:
ok, you're right